constants.js

```diff
@@ -100,6 +102,7 @@ const checksWhitelist = {
     [checkCompletedEvent]: [ciFailureCheck],
     [issueCommentCreatedEvent]: [respondToReviewCheck]
   },
+  // eslint-disable-next-line quote-props
```

Comment:
I have Disabled Lint beacuse while commiting `eslint *.js ./spec/*.js ./lib/*.js actions/**/*.js --fix` runs causing all auto fixable problems to be fixed.